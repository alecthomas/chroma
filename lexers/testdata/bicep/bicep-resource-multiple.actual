@batchSize(3)
resource sa 'Microsoft.Storage/storageAccounts@2019-06-01' = [for storageName in storageAccounts: {
  name: storageName
  location: location
  sku: {
    name: storageSKU
  }
  kind: 'StorageV2'
  properties: {
    supportsHttpsTrafficOnly: true
  }
}]