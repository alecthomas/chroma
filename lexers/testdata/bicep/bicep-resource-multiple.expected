[
  {"type":"Punctuation","value":"@"},
  {"type":"NameDecorator","value":"batchSize"},
  {"type":"Punctuation","value":"("},
  {"type":"Text","value":"3"},
  {"type":"Punctuation","value":")"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"KeywordDeclaration","value":"resource"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"sa"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralString","value":"'Microsoft.Storage/storageAccounts@2019-06-01'"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"="},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"["},
  {"type":"Keyword","value":"for"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"storageName"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Keyword","value":"in"},
  {"type":"TextWhitespace","value":" "},
  {"type":"NameProperty","value":"storageAccounts"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"name"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"storageName"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"location"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"location"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"sku"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n    "},
  {"type":"NameProperty","value":"name"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"storageSKU"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"Punctuation","value":"}"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"kind"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralStringInterpol","value":"'StorageV2'"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"NameProperty","value":"properties"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Punctuation","value":"{"},
  {"type":"TextWhitespace","value":"\n    "},
  {"type":"NameProperty","value":"supportsHttpsTrafficOnly"},
  {"type":"Punctuation","value":":"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Text","value":"true"},
  {"type":"TextWhitespace","value":"\n  "},
  {"type":"Punctuation","value":"}"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":"}]"}
]
